tests/tests/src/UntaggedVariants.mjs

```diff
@@ -386,9 +386,9 @@ function check$1(s) {
       return;
     }
     let match = s[0];
-    if (match === undefined || match === null || match === true) {
+    if (match === true) {
       let match$1 = s[1];
-      if (match$1 === undefined || match$1 === null || match$1 === false) {
+      if (match$1 === false) {
         let match$2 = s[2];
         if (match$2 === undefined || match$2 === null || match$2 === false || match$2 === true) {
           console.log("Nope...");
```

Comment:
Can't comment further below for some reason, and probably out of scope of this PR, but

```js
          if (match$3 === undefined || match$3 === null || match$3 === false || match$3 === true) {
            console.log("Nope...");
            return;
          }
          if (typeof match$3 === "string" && match$3 === "My name is") {
```

could just be

```js
          if (match$3 === "My name is") {
```

?

Reply:
can you move this to an issue? after this PR
to keep things from growing too much